source/dotnet/wholesale-api/Events/Events.Infrastructure/InboxEvents/AggregatedTimeSeriesMessageFactory.cs

```diff
@@ -17,19 +17,34 @@
 using Energinet.DataHub.Wholesale.Events.Application.InboxEvents;
 using Google.Protobuf;
 using Google.Protobuf.WellKnownTypes;
+using NodaTime.Serialization.Protobuf;
+using AggregatedTimeSeriesRequestContract = Energinet.DataHub.Edi.Requests.AggregatedTimeSeriesRequest;
+using PeriodContract = Energinet.DataHub.Edi.Responses.Period;
+using TimeSeriesTypeContract = Energinet.DataHub.Edi.Responses.TimeSeriesType;
 
 namespace Energinet.DataHub.Wholesale.Events.Infrastructure.InboxEvents;
 
 public class AggregatedTimeSeriesMessageFactory : IAggregatedTimeSeriesMessageFactory
 {
+    public AggregatedTimeSeriesRequest GetAggregatedTimeSeriesRequest(ServiceBusReceivedMessage request)
```

Comment:
To me factories create stuff. So if this is factory I'd rename it like this (possibly including all the private methods):
```suggestion
    public AggregatedTimeSeriesRequest CreateRequest(ServiceBusReceivedMessage request)
```
Note, I would also shorten the name as what it creates is already given from the type (class name).

Reply:
Good point, then I would split it into two classes instead :)